### ISSUE TITLE
optional pattern

### ISSUE THREAD

**Issue by smbache** [krlmlr/import.gen#3]
how about having an optional pattern to filter what is included?

```
import.gen::from("ggplot2", pattern = "element_.*")
```

**Comment by krlmlr** [comment]
The function is vectorized, it would have to be

```
import.gen::from(c("ggplot2" = "element_", "testthat" = "expect_"))
```

But I like the idea.

I guess exclusions can be solved with negative lookahead, if necessary at all.

**Comment by smbache** [comment]
yeah, exclusions could also be useful. Neg. look ahead is available with `perl = TRUE` AFAIK. 
Not sure what the "prettiest" interface would be...
